### ISSUE TITLE
log pop-up window not wide enough (feature request)

### ISSUE THREAD

**Issue by ericpar** [RapidSVN/RapidSVN#9]
The log modal window is not wide enough and manual stretching of the window is required every time we need to inspect the log.
See the attacheed file which is a screenshot.
![rapidsvn_log_window](https://cloud.githubusercontent.com/assets/1882164/19480721/73a60220-9519-11e6-8f91-e5d7cf25c1be.png)

A wonderful feature would be to save the pop-up's size for the next similar operation.

**NOTE**: I'm not familiar with RapidSVN's codebase but I would be happy to have a look at it and eventually submit a PR with a proposed modification. _I would appreciate a hint on where to look for that implementing such change_ (what directory or which files).

**Comment by davidcallen** [comment]
I agree this would be useful. I have a patch for this, although I simply size the dialog to same as the parent window (actually 5% smaller so can still see parent). This makes it a usable size;  it is simpler to code; and dont have to worry about if saved dimensions no longer are applicable to current situation of parent window.
Am attaching my git diff file.  
Am new to git (I know subversion!) so I haven't worked out how to push this commit yet - any advice on git workflow appreciated.
[git-diff-issue-9-fix.txt](https://github.com/RapidSVN/RapidSVN/files/544771/git-diff-issue-9-fix.txt)

**Comment by davidcallen** [comment]
Also noticed on linux the annotate dialog appears but is very small.  Applied similar code change to size this relative to parent window.
[git-diff-issue-9-fix-annotate-dlg.txt](https://github.com/RapidSVN/RapidSVN/files/548352/git-diff-issue-9-fix-annotate-dlg.txt)

**Comment by davidcallen** [comment]
Thanks eric. Apologies for just missing your push commit with my follow-on patch.

**Comment by davidcallen** [comment]
I have raised a Pull Request for this 2nd patch file from my github branch "davidcallen/RapidSVN". Hopefully this is correct procedure.
